rotkehlchen/premium/premium.py

```diff
@@ -178,8 +180,10 @@ def __init__(self, credentials: PremiumCredentials):
         self.session.mount('https://', adapter)
         self.apiversion = '1'
         self.rotki_api = f'https://rotki.com/api/{self.apiversion}/'
+        self.rotki_web = f'https://rotki.com/webapi/{self.apiversion}/'
```

Comment:
Did not even notice it was localhost before. What I said in the comment still stands. Did you actually agree on this api url endpoint for rotki web? Is this what we discussed with kelsos and agreed on? This is the rotkehlchen-web api you said would be needed right?

Reply:
we didn't talk about this with kelsos. The webapi is something that maps in traefik to django api so in theory should be fine but we can change it if needed when kelsos get back